code/game/machinery/computer/_computer.dm

```diff
@@ -159,7 +159,7 @@
 		if("edit_field")
 			target = locate(params["ref"]) in GLOB.manifest.general
 			var/field = params["field"]
-			if(!field || !target?.vars[field])
+			if(!field || !(field in target?.vars))
```

Comment:
So why exactly do the manifest ui_act actions exist on the base computer definition?

Reply:
Just shared between medical records and security records. The actions aren't used by any other console

Reply:
I could subtype further... or just copy the code between the two, if you want

Reply:
I just find it odd to find this behavior on the base type and not in a subtype because the base type doesn't have any of this functionality

Reply:
I'm not a maintainer so I can't tell you to do this or not to, it's just a request

Reply:
rules are meant to be broken

Reply:
put you as a coauthor on the last commit and it just assumes you made it...... ugh. anyway, let me know what you think.